bpf/bpf_host.c

```diff
@@ -1559,7 +1559,9 @@ int cil_to_netdev(struct __ctx_buff *ctx)
 	{
 		void *data, *data_end;
 		struct iphdr *ip4;
-		struct ipv4_ct_tuple tuple = {};
```

Comment:
This code path handles the pod-to-world direction, on the worker node. Matching against EGW policy, and potentially redirecting (via VXLAN) to the gateway node.